packages/rich-text-editor/docs/RichTextEditor.stories.tsx

```diff
@@ -14,28 +20,49 @@ export default {
   },
 }
 
-export const Default = args => {
+export const Default: Story<RichTextEditorProps> = args => {
   const [rteData, setRTEData] = useState<EditorContentArray>([])
   return (
     <>
       <RichTextEditor
         value={rteData}
         onChange={data => setRTEData(data)}
-        {...args}
+        labelText={"Label"}
+        rows={3}
+        controls={[
+          { name: "bold", group: "inline" },
+          { name: "italic", group: "inline" },
+          { name: "underline", group: "inline" },
+          { name: "orderedList", group: "list" },
+          { name: "bulletList", group: "list" },
+        ]}
       />
     </>
   )
 }
 
 Default.storyName = "Default (Kaizen Demo)"
-Default.args = {
```

Comment:
Please bring these back in as well, they are useful for allowing people to play around with props in storybook.